doc/api/events.md

```diff
@@ -1166,6 +1166,68 @@ const emitter = new EventEmitter();
 setMaxListeners(5, target, emitter);
 ```
 
+## Class: `events.EventEmitterAsyncResource extends EventEmitter`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+Integrates `EventEmitter` with {AsyncResource} for `EventEmitter`s that
```

Comment:
I guess {AsyncResource} should be a link.

Reply:
It is a link: 
![screenshot of the generated web page of the docs showing the link](https://user-images.githubusercontent.com/14309773/146836087-423ed9f2-5e87-4447-b389-c8fa4ae20494.png)